v3/src/components/calculator/calculator-model.ts

```diff
@@ -5,7 +5,8 @@ import { kCalculatorTileType } from "./calculator-defs"
 export const CalculatorModel = TileContentModel
   .named("CalculatorModel")
   .props({
-    type: types.optional(types.literal(kCalculatorTileType), kCalculatorTileType)
+    type: types.optional(types.literal(kCalculatorTileType), kCalculatorTileType),
+    isUserResizable: false
```

Comment:
I know you haven't asked for review yet, but I think this should be registered with the component info rather than stored with the model given that it doesn't change from one tile instance to another.